File: 05. Percabangan/F. Floor dan Ceiling/f_floor_dan_ceiling.c
#include <stdio.h>
#include <math.h>
int main(){
  double N;
  scanf("%lf", &N);
  if (N >= 0){
    if (N > trunc(N))
      printf("%0.lf %0.lf\n", trunc(N), trunc(N) + 1);
    else
      printf("%0.lf %0.lf\n", N, N);
  }
  else{
    if (N < trunc(N))
      printf("%0.lf %0.lf\n", trunc(N) - 1, trunc(N));
    else
      printf("%0.lf %0.lf\n", N, N);
  }
}

File: 05. Percabangan/F. Floor dan Ceiling/f_floor_dan_ceiling.go
package main

import (
  "fmt"
  "math"
)

func main() {
  var N float64
  fmt.Scanf("%f", &N)
  if N >= 0 {
    if N > math.Trunc(N) {
      fmt.Printf("%.0f %.0f\n", math.Trunc(N), math.Trunc(N)+1)
    } else {
      fmt.Printf("%.0f %.0f\n", N, N)
    }
  } else {
    if N < math.Trunc((N)) {
      fmt.Printf("%.0f %.0f\n", math.Trunc(N)-1, math.Trunc(N))
    } else {
      fmt.Printf("%.0f %.0f\n", N, N)
    }
  }
}
